pkg/xds/bootstrap/types/bootstrap_request.go

```diff
@@ -14,18 +14,18 @@ type BootstrapRequest struct {
 	Host               string  `json:"-"`
 	Version            Version `json:"version"`
 	// CaCert is a PEM-encoded CA cert that DP uses to verify CP
-	CaCert               string                     `json:"caCert"`
-	DynamicMetadata      map[string]string          `json:"dynamicMetadata"`
-	DNSPort              uint32                     `json:"dnsPort,omitempty"`
-	ReadinessPort        uint32                     `json:"readinessPort,omitempty"`
-	AppProbeProxyEnabled bool                       `json:"appProbeProxyDisabled,omitempty"`
-	OperatingSystem      string                     `json:"operatingSystem"`
-	Features             []string                   `json:"features"`
-	Resources            ProxyResources             `json:"resources"`
-	Workdir              string                     `json:"workdir"`
-	MetricsResources     MetricsResources           `json:"metricsResources"`
-	SystemCaPath         string                     `json:"systemCaPath"`
-	TransparentProxy     *tproxy_dp.DataplaneConfig `json:"dataplaneConfig,omitempty"`
+	CaCert                string                     `json:"caCert"`
+	DynamicMetadata       map[string]string          `json:"dynamicMetadata"`
+	DNSPort               uint32                     `json:"dnsPort,omitempty"`
+	ReadinessPort         uint32                     `json:"readinessPort,omitempty"`
+	AppProbeProxyDisabled bool                       `json:"appProbeProxyDisabled,omitempty"`
```

Comment:
I think the only change you need to make is to change this json tag to align with the field name right?
The meaning of the fields is not confused/error right now, the issue is the json tag.

Reply:
Changing the jsonTag is a breaking change. DP and CP run different versions so you'd break our version compat.